### ISSUE TITLE
Breaks with multiple panes

### ISSUE THREAD

**Issue by erickbarron** [rovansteen/atom-relative-line-numbers#1]
I think this package is awesome, and it works great, it just breaks some of the lines when using multiple panes. It adds a number that shouldn't be there for "true number on current line" and also breaks the others a little. 

Windows 7, 64bit
Atom Version: 0.191.0

![relative-lines](https://cloud.githubusercontent.com/assets/3500392/7171691/9eaefee0-e390-11e4-86da-c47163432dd7.gif)

**Comment by rovansteen** [comment]
Thanks for reporting this bug, I'll look into it.

**Comment by rovansteen** [comment]
Are you using soft line breaks?

**Comment by erickbarron** [comment]
Yeah, I'm using 'Soft Wrap' if that's what you mean. Also using Vim-mode if that makes any difference.

**Comment by rovansteen** [comment]
Yes that's what I meant. I've had the same problems after enabling soft wrap. Can you confirm that if you disable soft wrap it's working fine? I'll add support for soft wraps in the next version then.

**Comment by erickbarron** [comment]
Yeah, I can confirm too. It seems to be soft wraps. Thanks!
